.github/workflows/firesim-run-tests.yml

```diff
@@ -161,6 +161,17 @@ jobs:
       - name: Check docs components that require manual re-generation (e.g. config_runtime.yaml example, help output)
         run: ./.github/scripts/check-docs-generated-components.py
 
+  build-f1-driver:
+    name: build-f1-driver
```

Comment:
Did you want to serialize this due to the SBT invocation?

Reply:
Yeah I did initially but I have a feeling since it always failed on test-group-A it might just have been the termination bridge issue. Maybe it's safer to serialize it again though

Reply:
I think you should to be safe.

Reply:
serialized it, should be good to go now